jina/types/sets/document_set.py

```diff
@@ -49,7 +49,7 @@ def __getitem__(self, item):
         if isinstance(item, int):
             return Document(self._docs_proto[item])
         elif isinstance(item, str):
-            return Document(self._docs_map[str(item)])
```

Comment:
this is required, as `UniqueId` can not be directly used as a string

Reply:
oh but the test doesn't complain 🤔

Reply:
![image](https://user-images.githubusercontent.com/9794489/100431269-32c70c80-3098-11eb-9c4a-9797b6d1a17b.png)
@hanxiao this is what I got in terminal